AntiCheat/Lang/en_US.resx

```diff
@@ -121,100 +121,100 @@
-    <value>Mini-Map mod detection</value>
+    <value>Detects players using a mini-map mod, displaying the radar on the screen at all times.</value>
   </data>
   <data name="config_Mask" xml:space="preserve">
-    <value>Abnormal masked enemy detection</value>
+    <value>Detects when masked enemies are spawned abnormally.</value>
   </data>
   <data name="config_Nameless" xml:space="preserve">
-    <value>Player name detection (Nameless or Unknown)</value>
+    <value>Detects invalid or anomalous player names such as Nameless.</value>
   </data>
   <data name="config_Prefix" xml:space="preserve">
-    <value>Set the lobby name prefix, empty to disable</value>
+    <value>Puts text in square brackets before the lobby name, leave empty to disable.</value>
   </data>
   <data name="config_RemoteTerminal" xml:space="preserve">
-    <value>Remote terminal detection</value>
+    <value>Detects when a player uses the terminal remotely.</value>
   </data>
   <data name="config_ShipBuild" xml:space="preserve">
-    <value>Abnormal furniture position detection</value>
+    <value>Detects the abnormal placement of furniture.</value>
   </data>
   <data name="config_ShipConfig2" xml:space="preserve">
-    <value>Client-side player can only pull the lever when specified player number count are reached (StartGameOnlyHost must set to false)</value>
+    <value>Clients can only pull the lever when this player count is reached. (StartGameOnlyHost must be set to false.)</value>
   </data>
   <data name="config_ShipConfig3" xml:space="preserve">
-    <value>Allow client-side player to take off when after this time (based on game world)</value>
+    <value>Clients can take off after this in-game time.</value>
   </data>
   <data name="config_ShipConfig4" xml:space="preserve">
-    <value>Allow client-side player to take off when the number of alive players in the ship are exceed the specified ratio of total alive players number (0-100%)</value>
+    <value>Clients can take off when the number of living players in the ship is equal or greater than the specified percentage. (0-100%)</value>
   </data>
   <data name="config_ShipConfig5" xml:space="preserve">
-    <value>Detect malicious lever pull behavior (kick player when this behavior is detected)</value>
+    <value>Detects when a player attempts to force the ship lever. Kicks the player if detected.</value>
   </data>
   <data name="config_ShipConfig6" xml:space="preserve">
-    <value>You are the host, your vote can call the autopilot.</value>
+    <value>Allow your single vote to instantly force the autopilot as host.</value>
   </data>
   <data name="config_ShipLight" xml:space="preserve">
-    <value>Spamming light switch detection</value>
+    <value>Detects when a player spawns a light switch.</value>
   </data>
   <data name="config_ShipSetting" xml:space="preserve">
-    <value>Only the host can land the ship</value>
+    <value>Makes it so that only the host may land the ship.</value>

```

Comment:
VERY ONLY the host can land the ship

Reply:
I'm not sure I understand what the correction is here. The wording is what you said--it says that with this setting enabled, only the host can land the ship.

Reply:
hmm changed to this is fine actually, ok